gradle.properties

```diff
@@ -20,6 +20,6 @@ findbugsWarningThreshold = 0
 pmdErrorThreshold = 249
 pmdWarningThreshold = 463
 
-cpdErrorThreshold = 337
+cpdErrorThreshold = 354
```

Comment:
This raise is valid. Introducing new probes also introduces additional duplicate code.